cluster/ci/monitoring/mixins/config.libsonnet

```diff
@@ -21,6 +21,11 @@
         channel: '#build-cop-alerts',
         notify: 'build-cop',
       },
+      'openshift-library': {
+        team: 'openshift-library',
```

Comment:
the team here is the developer experience team.  not sure if this is supposed to correlate to a specific value (e.g. slack alias) or is just free form?

Reply:
Free form.
I will update accordingly.

Reply:
Done.